rfd/0037-desktop-access-protocol.md

```diff
@@ -255,3 +263,17 @@ Per-session MFA for desktop access works the same way as it does for SSH
 sessions. A JSON-encoded challenge is sent over websocket to the user's browser.
 The only difference is that SSH sessions wrap the MFA JSON in a protobuf
 encoding, where desktop sessions wrap the MFA JSON in a TDP message.
+
+#### 32 - sync keys
+
+```
+| message type (32) | caps_lock_state byte |
+```
+
+This message synchronizes the state of keyboard's modifier keys (caps lock)
+and resets the server key state to all keys up.
+
+`caps_lock_state` is one of:
+
+- `0` for caps lock inactive
+- `1` FOR CAPS LOCK ACTIVE
```

Comment:
It's okay if we don't implement other toggle keys, but we should probably include them in the TDP message right away.